src/Security/Security/Cmdlets/AllowedConnections/GetAllowedConnections.cs

```diff
@@ -0,0 +1,71 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ------------------------------------
+using System.Linq;
+using System.Management.Automation;
+using Commands.Security;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.Security.Common;
+using Microsoft.Azure.Commands.Security.Models.AllowedConnections;
```

Comment:
looks like this allowedconnections was not defined yet

Reply:
Sorry about that. I will remove it.

Reply:
Removed.